lib/ci/travis.js

```diff
@@ -24,10 +24,11 @@ class Travis extends CiBase {
    * @returns {Promise} a promise resolved with the result of the push
    */
   push () {
+    const branch = this.config.branch
     return this.vcs.addRemoteForPush()
       .then((remoteName) => {
-        logger.log(`Pushing my-master to ${remoteName}`)
-        return exec(`git push ${remoteName} my-master:refs/heads/master --tags`)
+        logger.log(`Pushing ci-${branch} to ${remoteName}`)
+        return exec(`git push ${remoteName} ci-${branch}:refs/heads/${branch} --tags`)
```

Comment:
I just thought it was more specific that this is something the `ci` is doing, it's really just arbitrary, but since I had `my-branch` as an example of a branch name, I didn't like having `my-my-branch` ;)